inference-engine/include/cpp/ie_cnn_network.h

```diff
@@ -54,6 +54,8 @@ class INFERENCE_ENGINE_API_CLASS(CNNNetwork) {
      * @param network Pointer to the ngraph::Function object
      */
     explicit CNNNetwork(const std::shared_ptr<const ngraph::Function>& network);
+    explicit CNNNetwork(const std::shared_ptr<ngraph::Function>& network);
```

Comment:
BTW, if user create network using ngraph code, then pass to `CNNNetwork`, it seems new ctor will be used (which allows to change a network object). So, the behavior is changed?

Reply:
In general CNNNetwork already has `getFunction()` method which allows to modify nGraph function.
I only added constructor which allow to avoid redundant copy.

We strictly forbidden to change nGraph function when we had a problem with network syncronisation.

Reply:
Ok, why do we need to have:
```cpp
explicit CNNNetwork(const std::shared_ptr<const ngraph::Function>& network);
```
in this case?

Reply:
In user want to avoid changes in the original network, he can choose this way

Reply:
but as you said - `getFunction` allows to change a network anyway

Reply:
let's add explicit words what this ctors just wraps existing `ngraph::Function`

Reply:
Updated docs